src/js/news-cards.js

```diff
@@ -0,0 +1,134 @@
+  </svg>
+  <svg class="list-news-card__add-favorite--svg color-svg2 hidden" width="13" height="12" viewBox="0 0 32 32">
+  <path d="M8.382 2.286C4.174 2.286.761 5.662.761 9.829c0 3.362 1.335 11.344 14.459 19.413a1.494 1.494 0 0 0 1.565-.004l-.006.004c13.125-8.069 14.459-16.05 14.459-19.413 0-4.167-3.413-7.543-7.621-7.543-4.206 0-7.618 4.571-7.618 4.571s-3.413-4.571-7.618-4.571z"/>
+</svg>
+</button>
+    <div class="container-news-list__date-read"><span class="list-news-card__newsDate">${updated}</span>
+  <a href="${url}" class="list-news-card__link-read-more" target="_blank" data-link='link'>Read more</a></div>
+</li>`;
+}
+function updateNews(markup) {
+    ulEl.insertAdjacentHTML('beforeend', markup);
+}
+function onError(error) {
+    console.log(error);
+}
+
+ulEl.addEventListener('click', onBtnClick);
+
+function onBtnClick(e) {
+    e = e.target;
+
+    let btn = e.parentNode;
+
+    if (btn.type !== 'button') {
+        return;
+    }
+
+    if (btn.dataset.add === 'true') {
+        //Значит новость не добавлена и нужно ее добавить
+        btn.firstElementChild.textContent = 'Remove from favorite';
+        btn.dataset.add = false;
+        btn.firstElementChild.nextElementSibling.classList.add('hidden');
+        btn.firstElementChild.nextElementSibling.nextElementSibling.classList.remove(
+            'hidden'
+        );
+        btn.firstElementChild.nextElementSibling.nextElementSibling.classList.add(
+            'color-svg2'
+        );
+
+        const parent = e.closest('li');
+        console.dir(parent.dataset);
+        const toSave = { ...parent.dataset, isFavorite: true, isRead: false };
+
+        localStorage.addToFavorites(toSave);
+    } else {
+        //Значит новость добавлена и нужно ее удалить
+        btn.firstElementChild.textContent = 'Add to favorite';
+        btn.dataset.add = true;
+        btn.firstElementChild.nextElementSibling.classList.remove('hidden');
+        btn.firstElementChild.nextElementSibling.nextElementSibling.classList.add(
+            'hidden'
+        );
+        const parent = e.closest('li');
+        console.dir(parent.dataset);
+        const toDel = {
+            ...parent.dataset,
+            isFavorite: true,
+            isRead: false,
+        };
+        localStorage.deleteFromFavorites(toDel);
+    }
+}
+
+ulEl.addEventListener('click', onLinkClick);
+
+function onLinkClick(event) {
+    event.preventDefault();
+    const link = event.target;
+
+    if (link.dataset.link !== 'link') {
+        return;
+    }
+
+    const parent = link.closest('li');
+    console.dir(parent.dataset);
+    const toSave = { ...parent.dataset, isFavorite: true, isRead: true };
+
+    // if (!toSave.isRead) {
+    //     return;
+    // }
+    link.parentNode.parentNode.classList.add('opacity');
+    localStorage.addToReaded(toSave);
```

Comment:
Перейменуй, будь ласка, цю функцію. 
```suggestion
    localStorage.addToRead(toSave);
```